pkg/domain/infra/abi/secrets.go

```diff
@@ -146,6 +149,7 @@ func (ic *ContainerEngine) SecretRm(ctx context.Context, nameOrIDs []string, opt
 			continue
 		}
 		reports = append(reports, &entities.SecretRmReport{Err: err, ID: deletedID})
+		ic.Libpod.NewSecretEvent(events.Remove, deletedID)
```

Comment:
This seems to generate an event even when there was an error deleting it, shouldn't this only be generated on err ==nil?

Reply:
True, good catch, thanks :-).